src/CoreWCF.Kafka/src/CoreWCF/Channels/KafkaMessageProperty.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using Confluent.Kafka;
+
+namespace CoreWCF.Channels;
+
+public sealed class KafkaMessageProperty : IMessageProperty
+{
+    private readonly IList<KafkaMessageHeader> _headers = new List<KafkaMessageHeader>();
+
+    public static readonly string Name = "kafkaMessage";
+
```

Comment:
`RemoteEndpointMessageProperty.Name` has value `"CoreWCF.Channels.RemoteEndpointMessageProperty"`, but `HttpRequestMessageProperty.Name` has value `"httpRequest"`. It's a bit erratic and inconsistent, but I like to try to avoid name clashes. When adding `HttpContext` to the message properties I used the name `"Microsoft.AspNetCore.Http.HttpContext"`. I think it's a good coding guideline that we should adopt that message property names should be the class full name. So in this case, `"CoreWCF.Channels.KafkaMessageProperty"`

Reply:
Done